tap_mixpanel/schemas/funnels.json

```diff
@@ -3,60 +3,105 @@
   "additionalProperties": false,
   "properties": {
     "funnel_id": {
-      "type": ["null", "integer"]
+      "type": [
+        "null",
+        "integer"
+      ]
     },
     "name": {
-      "type": ["null", "string"]
+      "type": [
+        "null",
+        "string"
+      ]
     },
     "date": {
-      "type": ["null", "string"],
+      "type": [
+        "null",
+        "string"
+      ],
       "format": "date"
     },
     "datetime": {
-      "type": ["null", "string"],
+      "type": [
```

Comment:
@namrata270998 Re-structuring the "type" array is not expected as part of this PR. Can you please revert this change?